spec/features/idv/doc_auth/hybrid_handoff_spec.rb

```diff
@@ -235,6 +253,27 @@
       end
     end
 
+    context 'with PhoneQuestion page disabled' do
+      before do
+        allow_any_instance_of(Idv::HybridHandoffController).
+          to receive(:phone_question_ab_test_bucket).and_return(:bypass_phone_question)
+
+        user = user_with_2fa
+        sign_in_and_2fa_user(user)
+        complete_doc_auth_steps_before_hybrid_handoff_step
+      end
+
+      it 'displays the expected headings' do
+        expect(page).not_to have_current_path(idv_phone_question_path)
+        expect(page).to have_current_path(idv_hybrid_handoff_path, ignore_query: true)
+
+        expect(page).to have_selector('h1', text: t('doc_auth.headings.hybrid_handoff'))
+        expect(page).to have_selector('h2', text: t('doc_auth.headings.upload_from_phone'))
+        expect(page).not_to have_selector('h1', text: t('doc_auth.headings.upload_from_phone'))
+        expect(page).not_to have_selector('h2', text: t('doc_auth.headings.switch_to_phone'))
```

Comment:
Thanks for adding these checks @night-jellyfish  ... is it possible these checks could be inserted into one of the already existing tests to reduce the time it takes for our long running spec?

Reply:
That's a good point. I struggled with that myself but I wasn't sure how to incorporate them into another spec and keep the reasoning for testing clear. I sometimes think about tests like documentation, and I liked that the `context` and `it` blocks let me communicate exactly what I'm testing / what I expect the code to do. It seems like it might help towards your other point of making the test easy to delete, as it's encapsulated.

But, I understand your point about test speed as well, as slow tests can cause problems and pain as well.

Have you found a way to merge the gap between those two pulls (clarity vs speed)?

My thoughts were, since there aren't other tests focused on copy, if I added it to a test, the `it` block would become a little strange, ex: `it 'rate limits sending the link and displays the expected headings'`, which aren't really related. But, maybe I'm overthinking it.

Reply:
Thank you for the feedback! As discussed over Slack, I agree that since it's just a text change, it's fine to move into the context of other test to reduce impact on test time. I've pushed up this change in [3f7b868](https://github.com/18F/identity-idp/pull/9316/commits/3f7b86843fbb7b2bb7f175f232e2df57b09eced0).

Since you approved, I'll merge if CI passes - but please let me know if you have any hesitancy on that!